### PR TITLE
Added new features to Top-Down PEP

### PR DESCRIPTION
**Pull request by nbollis**
Added the following features
- Peaks in precursor envelope
- Count of identifications per scan
- Intensity of most abundant precursor peak
- Fractional intensity of precursor in isolation window

Removed the normalization for top down PEP

Current Master is green
This PR is cyan
PrSM and Proteoform count are determined by those with <= 0.01 PEP_QValue

AllResults_TopDownSummary

### CODE REVIEW

File: MetaMorpheus/Test/FdrTest.cs [hunk 1]
```diff
@@ -639,16 +639,38 @@ public static void TestPsmData()
         {
             string searchType = "standard";
             string[] trainingInfoStandard = PsmData.trainingInfos[searchType];
-            string[] expectedTrainingInfoStandard = new[] { "TotalMatchingFragmentCount", "Intensity", "PrecursorChargeDiffToMode", "DeltaScore", "Notch", "ModsCount", "AbsoluteAverageFragmentMassErrorFromMedian", "MissedCleavagesCount", "Ambiguity", "LongestFragmentIonSeries", "ComplementaryIonCount", "HydrophobicityZScore", "IsVariantPeptide", "IsDeadEnd", "IsLoop", "SpectralAngle", "HasSpectralAngle" };
+            string[] expectedTrainingInfoStandard = new[]
```
Comment (nbollis) [inline MetaMorpheus/Test/FdrTest.cs:642]: No change, just spacing it our for readability

File: MetaMorpheus/Test/FdrTest.cs [hunk 2]
```diff
@@ -639,16 +639,38 @@ public static void TestPsmData()
...
+                "ModsCount", "AbsoluteAverageFragmentMassErrorFromMedian", "MissedCleavagesCount", "Ambiguity",
+                "LongestFragmentIonSeries", "ComplementaryIonCount", "HydrophobicityZScore", "IsVariantPeptide",
+                "IsDeadEnd", "IsLoop", "SpectralAngle", "HasSpectralAngle"
+            };
             Assert.AreEqual(expectedTrainingInfoStandard, trainingInfoStandard);
 
             searchType = "top-down";
             string[] trainingInfoTopDown = PsmData.trainingInfos[searchType];
-            string[] expectedTrainingInfoTopDown = new[] { "TotalMatchingFragmentCount", "Intensity", "PrecursorChargeDiffToMode", "DeltaScore", "Notch", "ModsCount", "AbsoluteAverageFragmentMassErrorFromMedian", "Ambiguity", "LongestFragmentIonSeries", "ComplementaryIonCount", "SpectralAngle", "HasSpectralAngle" };
+            string[] expectedTrainingInfoTopDown = new[]
```
Comment (nbollis) [inline MetaMorpheus/Test/FdrTest.cs:653]: 4 additional features for top down

File: MetaMorpheus/EngineLayer/FdrAnalysis/PsmData.cs [hunk 3]
```diff
@@ -9,9 +9,36 @@ public class PsmData
     {
         public static readonly IImmutableDictionary<string, string[]> trainingInfos = new Dictionary<string, string[]>
         {
-            { "standard", new [] { "TotalMatchingFragmentCount", "Intensity", "PrecursorChargeDiffToMode", "DeltaScore", "Notch", "ModsCount", "AbsoluteAverageFragmentMassErrorFromMedian", "MissedCleavagesCount", "Ambiguity", "LongestFragmentIonSeries", "ComplementaryIonCount", "HydrophobicityZScore", "IsVariantPeptide", "IsDeadEnd", "IsLoop", "SpectralAngle", "HasSpectralAngle" } },
-            { "top-down", new [] { "TotalMatchingFragmentCount", "Intensity", "PrecursorChargeDiffToMode", "DeltaScore", "Notch", "ModsCount", "AbsoluteAverageFragmentMassErrorFromMedian", "Ambiguity", "LongestFragmentIonSeries", "ComplementaryIonCount", "SpectralAngle", "HasSpectralAngle" } },
-            { "crosslink", new [] { "TotalMatchingFragmentCount", "AbsoluteAverageFragmentMassErrorFromMedian", "PrecursorChargeDiffToMode", "DeltaScore", "AlphaIntensity", "BetaIntensity", "LongestFragmentIonSeries_Alpha", "LongestFragmentIonSeries_Beta", "IsInter", "IsIntra" } }
+            {
+                "standard", new[]
```
Comment (nbollis) [inline MetaMorpheus/EngineLayer/FdrAnalysis/PsmData.cs:13]: No change. Spacing out for readability

File: MetaMorpheus/EngineLayer/FdrAnalysis/PsmData.cs [hunk 4]
```diff
@@ -9,9 +9,36 @@ public class PsmData
...
+                {
+                    "TotalMatchingFragmentCount", "Intensity", "PrecursorChargeDiffToMode", "DeltaScore",
+                    "Notch", "ModsCount", "AbsoluteAverageFragmentMassErrorFromMedian", "MissedCleavagesCount", 
+                    "Ambiguity", "LongestFragmentIonSeries", "ComplementaryIonCount", "HydrophobicityZScore",
+                    "IsVariantPeptide", "IsDeadEnd", "IsLoop", "SpectralAngle", "HasSpectralAngle", 
+                }
+            },
+
+            {
+                "top-down", new[]
```
Comment (nbollis) [inline MetaMorpheus/EngineLayer/FdrAnalysis/PsmData.cs:23]: 4 new top down features

File: MetaMorpheus/EngineLayer/FdrAnalysis/PsmData.cs [hunk 5]
```diff
@@ -9,9 +9,36 @@ public class PsmData
...
+                    "TotalMatchingFragmentCount", "Intensity", "PrecursorChargeDiffToMode", "DeltaScore", 
+                    "Notch", "ModsCount", "AbsoluteAverageFragmentMassErrorFromMedian", "Ambiguity", 
+                    "LongestFragmentIonSeries", "ComplementaryIonCount", "SpectralAngle",
+                    "HasSpectralAngle", "PeaksInPrecursorEnvelope", "ChimeraCount",
+                    "MostAbundantPrecursorPeakIntensity", "PrecursorFractionalIntensity", "InternalIonCount"
+                }
+            },
+
+            {
+                "crosslink",
```
Comment (nbollis) [inline MetaMorpheus/EngineLayer/FdrAnalysis/PsmData.cs:34]: No change. Spacing out for readability

**Review by Alexander-Sol — APPROVED** [review]

**Review by trishorts — APPROVED** [review]

**Review by RayMSMS — APPROVED** [review]

**Pull request merged**